manifests/base/params-latest.env

```diff
@@ -21,7 +21,7 @@ odh-workbench-jupyter-datascience-cpu-py312-ubi9-n=quay.io/modh/odh-workbench-ju
 odh-workbench-jupyter-pytorch-cuda-py312-ubi9-n=quay.io/modh/odh-workbench-jupyter-pytorch-cuda-py312-ubi9@sha256:99ee6befe1246e5b8d92684f67181f96af1e75281fbcb2833a59fda70066c921
 odh-workbench-jupyter-pytorch-rocm-py312-ubi9-n=quay.io/modh/odh-workbench-jupyter-pytorch-rocm-py312-ubi9@sha256:e77bf86a942617bd29a6b520526eaca90b0cbfcc32f8bea193dfc051a9f07c69
 odh-workbench-jupyter-tensorflow-cuda-py312-ubi9-n=quay.io/modh/odh-workbench-jupyter-tensorflow-cuda-py312-ubi9@sha256:e9630c43375182ece8ec34d51203e745a6f1f0a0a2b6e15c362f4edaa7cd3351
-odh-workbench-jupyter-trustyai-cpu-py312-ubi9-n=quay.io/modh/odh-workbench-jupyter-trustyai-cpu-py312-ubi9@sha256:sha256:70ebb1265df60d680e1e2a07ed590201237042d87eac6c1dc64440e2559effd8
+odh-workbench-jupyter-trustyai-cpu-py312-ubi9-n=quay.io/modh/odh-workbench-jupyter-trustyai-cpu-py312-ubi9@sha256:70ebb1265df60d680e1e2a07ed590201237042d87eac6c1dc64440e2559effd8
```

Comment:
Hey, what is happening here? This is the second time in less than 24 hours this double `sha256` occured. Is there some issue during the automation???? :thinking:  Ideally we should have this as a separate commit, though.

Reply:
It is human error by me 🤣 I am really sorry, In the morning I wanted to update the image SHAs to real SHAs on that file and I added accidentatly two times the `sha256`

Reply:
I saw this fixed by Moulali this morning, so I'm confused TBH. Let me find it: https://github.com/red-hat-data-services/notebooks/commit/b45524bf76b1a278026c5e60438d242b351bf268#diff-b6555635f90debd689f02ce3003de5204f29a4d562a01c619a3f61baf5ba75f7L29

Reply:
okay, so - the main branch wasn't updated and as such, the changes were propagated back to the rhoai-2.23 branch... understand, this is unfortunate